mpu6050/mpu6050_main.c

```diff
@@ -0,0 +1,373 @@
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/device.h>
+#include <linux/err.h>
+#include <linux/i2c.h>
+#include <linux/i2c-dev.h>
+#include <linux/sysfs.h>
+#include <linux/time.h>
+#include <linux/kthread.h>
+#include <linux/completion.h>
+
+#include "mpu6050-regs.h"
+#include "mpu6050_data.h"
+#include "mpu6050_cdev.h"
+
+/* data */
+#define ELEMENTS_COUNT 10
+
+/* functionality */
+static size_t get_attribute_index(struct kobj_attribute const *attribute);
+
+static int mpu6050_read_data(bool debug)
```

Comment:
This is definitely wrong way of controlling debug printout.
It should be done on preprocessing stage.
Consider defining macro wrapper over `dev_info`, which will be expanded to nothing if debug mode not defined.

Reply:
That is over-control. It used to allow do not print for any type: or debug, or release

Reply:
Renaming the argument to `print_out` doesn't change anything.
The value provided to it is hardcodded, so there's no sense to keep it as parameter - define should be used instead.